poker-pals/src/main.js

```diff
@@ -1,4 +1,15 @@
 import Vue from 'vue'
+
+import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
+
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+// Install BootstrapVue
+Vue.use(BootstrapVue)
```

Comment:
This is using the component that you've imported, not installing.

Reply:
Oh i just copied the code from the bootstrap vue instructions

Reply:
I think what that comment meant was probably installing the plugin, not the package itself. Currently, I would interpret the comment as installing the package instead of the plugin.